book/java-rebuilds-service.md

```diff
@@ -0,0 +1,154 @@
+# JVM Build Service
```

Comment:
The file is called `java-rebuilds-service` while the title is `JVM Build Service`. Is one of these more accurate?

Reply:
JVM is more accurate, I will update the file name.